src/accounts/nexus.ts

```diff
@@ -177,10 +177,11 @@ async function getPackedSignature(
   hash: Hex,
   validator: ValidatorConfig,
   transformSignature: (signature: Hex) => Hex = (signature) => signature,
+  defaultValidatorAddress: Address = NEXUS_DEFAULT_VALIDATOR_ADDRESS,
 ) {
   const signature = await signFn(hash)
   const validatorAddress =
-    validator.address === NEXUS_DEFAULT_VALIDATOR_ADDRESS
+    validator.address === defaultValidatorAddress
       ? zeroAddress
```

Comment:
We add the optional override for the Nexus's default validator address, so that Startale can specify the original K1 validator, and the new Nexus can use the Ownable validator.

This lets use to reuse most of the Nexus code for Startale.